docs/content/latest/explore/ysql-language-features/data-manipulation.md

```diff
@@ -0,0 +1,343 @@
+---
+title: Data Manipulation
+linkTitle: Data Manipulation
+description: Data Manipulation in YSQL
+headcontent: Data Manipulation
+image: /images/section_icons/secure/create-roles.png
+menu:
+  latest:
+    identifier: explore-ysql-language-features-data-manipulation
+    parent: explore-ysql-language-features
+    weight: 200
+isTocNested: true
+showAsideToc: true
+---
+
+This section describes how to manipulate data in YugabyteDB using the YSQL `INSERT`, `UPDATE`, and `DELETE` statements.
+
+## Inserting Rows
+
+Initially, database tables are not populated with data. Using YSQL, you can add one or more rows containing comple or partial data by inserting one row at a time. 
+
+For example, you work with a database that includes the following table:
+
+```sql
+CREATE TABLE employees (
+    employee_no integer,
+    name text,
+    department text
+);
+```
+
+Assuming you know the order of columns in the table, you can insert a row by executing the following command:
+
+```sql
+INSERT INTO employees VALUES (1, 'John Smith', 'Marketing');  
+```
+
+If you do not know the order of columns, you have an option of listing them within the `INSERT`  statement when adding a new row, as follows: 
+
+```sql
+INSERT INTO employees (employee_no, name, department) VALUES (1, 'John Smith', 'Marketing');  
+```
+
+### Default Values
+
+In some cases you might not know values for all the columns when you insert a row. You have an option of not specifying these values at all, in which case the columns are automatically filled with default values when the `INSERT`  statement is executed, as demonstartrated in the following example:
+
+```sql
+INSERT INTO employees (employee_no, name) VALUES (1, 'John Smith');  
+```
+
+Another option is to explicitly specify the missing values as  `DEFAULT`  in the `INSERT`  statement, as shown in the following example:
+
+```sql
+INSERT INTO employees (employee_no, name, department) VALUES (1, 'John Smith', DEFAULT);  
+```
+
+### Multiple Rows
+
+You can use YSQL to insert multiple rows by executing a single `INSERT`  statement, as shown in the following example:
+
+```sql
+INSERT INTO employees (employee_no, name, department) 
+VALUES 	
+(1, 'John Smith', 'Marketing'),
+(2, 'Betty Davis', 'Sales'),
+(3, 'Lucille Ball', 'Operations'); 
+```
+
+### Upsert
+
+Upsert is a merge during a row insert: when you insert a new table row, YSQL checks if this row already exists, and if so, updates the row; otherwise, a new row is inserted.
+
+YSQL provides the `INSERT ON CONFLICT` statement that you can use to perform upserts, as follows:
+
+```sql
+INSERT INTO employees (employee_no, name, department) 
+VALUES (1, 'John Smith', 'Sales')
+ON CONFLICT (department) 
+DO NOTHING;
+```
```

Comment:
Added the following to the "Upsert" section: _In the preceding example, since no actions are required during merge of John Smith's department, upsert does not cause any changes to the `department` column. _